### PR TITLE
One Script Multi Tgen

### PR DESCRIPTION
**Pull request by Rangababu-R**

### CODE REVIEW

File: scripts/udp_bidirectional_flow.py [hunk 1]
```diff
@@ -0,0 +1,138 @@
...
+    eth2, ip2, udp2 = f2.packet[0], f2.packet[1], f2.packet[2]
+
+    # set source and destination MAC addresses
+    eth1.src.value, eth1.dst.value = "00:AA:00:00:04:00", "00:AA:00:00:00:AA"
+    eth2.src.value, eth2.dst.value = "00:AA:00:00:00:AA", "00:AA:00:00:04:00"
+
+    # set source and destination IPv4 addresses
+    ip1.src.value, ip1.dst.value = "10.0.0.1", "10.0.0.2"
+    ip2.src.value, ip2.dst.value = "10.0.0.2", "10.0.0.1"
+    ip1.protocol.value = 17
```
Comment (ashutshkumr) [inline scripts/udp_bidirectional_flow.py:64]: add a TODO for why this is like so.

File: scripts/udp_bidirectional_flow.py [hunk 2]
```diff
@@ -0,0 +1,138 @@
...
+    This script does following:
+    - Send 1000 packets back and forth between the two ports at a rate of
+      1000 packets per second.
+    - Validate that total packets sent and received on both interfaces is as
+      expected using port metrics.
+    - Validate that captured UDP packets on both the ports are as expected.
+    """
+    # create a new API instance where location points to controller
+    # location = "https://<tgenip>:<443(or)11009(or)port_set_by_user>"
+    # ext = ixnetwork (or) none for Athena
```
Comment (ashutshkumr) [inline scripts/udp_bidirectional_flow.py:16]: do not mention athena anywhere. use `ixia-c` instead.
moreover, simplify the comments by just specifying what should be the location and ext for `ixia-c`, `ixnetwork` and `trex`

feel free to leave out the input for trex as blank

File: scripts/udp_bidirectional_flow.py [hunk 3]
```diff
@@ -0,0 +1,138 @@
...
+    api = snappi.api(location="https://localhost:443", verify=False, ext=None)
+    # and an empty traffic configuration to be pushed to controller later on
+    cfg = api.config()
+
+    # add two ports where location points to traffic-engine (aka ports)
+    # port.location = [athena] localhost:5555 or [ixnetwork] \
```
Comment (ashutshkumr) [inline scripts/udp_bidirectional_flow.py:22]: simplify here too. please don't use any notations like [ixnetwork], etc.

**Review by ashutshkumr — CHANGES_REQUESTED** [review]
need to simplify comments

File: scripts/udp_bidirectional_flow.py (hunk 2 above)
Comment (Rangababu-R) [reply scripts/udp_bidirectional_flow.py:16]: # create a new API instance where location points to controller
    # ixia-c, trex:
    #   location = "https://<tgen-ip>:<port>"
    # ixnetwork
    #   location = "https://<tgen-ip>:<port>", ext="ixnetwork"

File: scripts/udp_bidirectional_flow.py (hunk 3 above)
Comment (Rangababu-R) [reply scripts/udp_bidirectional_flow.py:22]: # add two ports where location points to traffic-engine (aka ports)
    # ixia-c, trex:
    #   port.location = "localhost:5555"
    # ixnetwork:
    #   port.location = "<chassisip>;card;port"
    #

File: scripts/udp_bidirectional_flow.py (hunk 2 above)
Comment (Rangababu-R) [reply scripts/udp_bidirectional_flow.py:16]: is it fine with the above way?

File: scripts/udp_bidirectional_flow.py (hunk 3 above)
Comment (Rangababu-R) [reply scripts/udp_bidirectional_flow.py:22]: is it fine with the above way?

**Pull request merged**